src/bundle/Resources/views/admin/section/view.html.twig

```diff
@@ -57,13 +57,15 @@
             <td>{{ section.identifier }}</td>
             <td>{{ section.id }}</td>
             <td class="text-right">
-                <a href="{{ path('ezplatform.section.update', {'sectionId': section.id}) }}"
-                   title="{{ 'section.edit'|trans|desc('Edit') }}"
-                   class="btn btn-icon mx-3"{% if not can_edit %} data-disabled{% endif %}>
-                    <svg class="ez-icon ez-icon-edit">
-                        <use xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#edit"></use>
-                    </svg>
-                </a>
+                {% if can_edit %}
+                    <a href="{{ path('ezplatform.section.update', {'sectionId': section.id}) }}"
```

Comment:
Please, be consistent with code formatting. Personally, I'd be glad if you stick to the styling format like there: https://github.com/ezsystems/ezplatform-admin-ui/pull/457/files#diff-e4560f2bcdf14fd5c741bb046887a7c5R80